packages/plugin/tests/avoid-typename-prefix.spec.ts

```diff
@@ -5,28 +5,22 @@ const ruleTester = new GraphQLRuleTester();
 
 ruleTester.runGraphQLTests('avoid-typename-prefix', rule, {
   valid: [
-    {
-      code: /* GraphQL */ `
-        type User {
-          id: ID!
-        }
-      `,
-    },
-    {
-      code: /* GraphQL */ `
-        interface Node {
-          id: ID!
-        }
-      `,
-    },
-    {
-      code: /* GraphQL */ `
-        type User {
-          # eslint-disable-next-line
-          userId: ID!
-        }
-      `,
-    },
+    /* GraphQL */ `
```

Comment:
it's nice to have [test-case-shorthand-strings](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/master/docs/rules/test-case-shorthand-strings.md) rule enabled